notes/2019-04-05-kick-off-week/README.md

```diff
@@ -117,7 +129,7 @@ We wrapped up this session with a couple standing questions to be addressed the
 ## Roadmap & Continuity
 
 In 2016, DWeb Summit involved mostly builders of technologies. 
-In 2018, participants represented multiple disciplines and efforts were made to broaden gender diversity.
+In 2018, participants represented multiple disciplines and efforts were made to broaden gender, ethnic and geographic diversity.
 In organizing the third event, our first Camp, we hope to more deeply explore who is represented and create more meaningful representations by curating for truly inclusive spaces.
 With increased emphasis on decentralized physical infrastructures, we bring whole new discussion threads about infrastructural governance and "below-IP" projects and protocols that have established communities in the Global South.
 We are transitioning to open planning processes and exploring use of open source tools, drawing inspirations from events large and small from ScuttleCamp and Our Networks to Chaos Computer Camp and Burning Man.
```

Comment:
drawing inspirations from events large and small from ScuttleCamp and Our Networks to Chaos Computer Camp and Burning Man. TWO SMALL CHANGES:  inspiration (not inspirations) and "Chaos Communication Camp"

Reply:
```suggestion
We are transitioning to open planning processes and exploring use of open source tools, drawing inspiration from events large and small from ScuttleCamp and Our Networks to Chaos Communication Camp and Burning Man.
```